collivery.php

```diff
@@ -1,18 +1,18 @@
 <?php
 
 define('_MDS_DIR_', __DIR__);
-define('MDS_VERSION', '3.1.8');
+define('MDS_VERSION', '3.1.11');
 include 'autoload.php';
 
 /*
  * Plugin Name: MDS Collivery
  * Plugin URI: https://collivery.net/integration/woocommerce
  * Description: Plugin to add support for MDS Collivery in WooCommerce.
```

Comment:
What about the docs, the supported version is also listed there?

Reply:
Thanks, will update there as well